### PR TITLE
Update AWS SDK to 3.5.0

### LINKED ISSUE #768: Update to AWS SDK 3.5
This might be a good time to take the update.
https://aws.amazon.com/blogs/developer/aws-sdk-for-net-v3-5-now-generally-available/

Old:
image
New:
image

#### FuGet diffs:
https://www.fuget.org/packages/AWSSDK.SQS/3.5.0.5/lib/netstandard2.0/diff/3.3.103.26/
https://www.fuget.org/packages/AWSSDK.SimpleNotificationService/3.5.0/lib/netstandard2.0/diff/3.3.102.17/
https://www.fuget.org/packages/AWSSDK.Core/3.5.0/lib/netstandard2.0/diff/3.3.107.40/

### Pros
- Fewer assemblies for legacy runtimes (smaller nuget downloads)
- Extra async enumerable APIs available to take advantage of potentially in future

### Cons
- Required version uplift for consumers
- Low risk breaking changes exist in `AWSSDK.Core` (If consumers are using "Pubternal" APIs 🤷‍♀️)

### PR DESCRIPTION
**Pull request by slang25**
Fixes #768

### CODE REVIEW

File: src/JustSaying/JustSaying.csproj [hunk 1]
```diff
@@ -6,8 +6,8 @@
     <ProjectReference Include="..\JustSaying.Models\JustSaying.Models.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.3.100.1" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.3.100.1" />
+    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.5.0" />
+    <PackageReference Include="AWSSDK.SQS" Version="3.5.0" />
```
Comment (slang25) [inline src/JustSaying/JustSaying.csproj:10]: I presume this is deliberately different from `AwsSdkSqsVersion` as this is intended to be the minimum supported version?

File: src/JustSaying/AwsTools/MessageHandling/SnsPolicy.cs [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#pragma warning disable 618
```
Comment (slang25) [inline src/JustSaying/AwsTools/MessageHandling/SnsPolicy.cs:1]: `Amazon.Auth` is now obsolete: https://github.com/aws/aws-sdk-net/commit/6f9f7aa20e92180ecbf062d8fba8f67dbcc68add

File: src/JustSaying/JustSaying.csproj (hunk 1 above)
Comment (martincostello) [reply src/JustSaying/JustSaying.csproj:10]: Yeah I "pinned" this for the libraries, but the other is for the tests to use "latest"

**Comment by martincostello** [comment]
If you rebase onto #781 once that's merged then we should be able to merge this 👍

**Comment by slang25** [comment]
My fork just re-parented to brian 😕 
https://github.com/justeat/JustSaying/issues/782

**Comment by martincostello** [comment]
@slang25 See Slack 😃

**Comment by slang25** [comment]
@martincostello that's an ongoing problem for me, I have my access revoked for most of the last 2 weeks, including right now 😆

**Pull request closed without merging**